protocol-designer/src/pages/Designer/ProtocolSteps/StepForm/StepTools/ByVolumeBuilderModal/styles.module.css

```
@@ -0,0 +1,29 @@
+.calculator_button {
+  width: 100%;
+  height: 'min-content';
```

i think this doesn't need to be a string

```suggestion
  height: min-content;
```